pkgs/by-name/ra/rapidjson/package.nix

```diff
@@ -74,6 +75,10 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "http://rapidjson.org/";
     license = licenses.mit;
     platforms = platforms.unix;
-    maintainers = with maintainers; [ dotlambda Madouura tobim ];
```

Comment:
suggestion: remove the `with lib;` from meta. and add `lib.` for license, platforms and maintainers